packages/zmarkdown/__tests__/server.tests.js

```diff
@@ -132,6 +132,16 @@ describe('LaTeX endpoint', () => {
     expect(metadata.disableToc).toBe(false)
   })
 
+  it('escapes title and author', async () => {
+    const response = await a.post(latex,
```

Comment:
Take a look at what you commit. The snapshot shows that you are not testing your changes. I guess the endpoint used on this line is not the right one.

Reply:
you seem right.